### PR TITLE
feat(route): 添加Binance数字货币及交易对上新公告RSS

### PR DESCRIPTION
**Pull request by zhenlohuang**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/binance/launchpool
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [ ] New Route / 新的路由
  - [ ] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [ ] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [ ] Parsed / 可以解析
  - [ ] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/routes/binance/launchpool.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    });
+
+    const $ = load(response.data);
+    const appData = JSON.parse($('#__APP_DATA').text());
+    const articles = appData.appState.loader.dataByRouteId.d969.catalogs.find((catalog) => catalog.catalogId === 48).articles.filter((article) => article.title.includes('币安新币挖矿上线'));
+
+    return {
+        title: 'Binance | 数字货币及交易对上新',
+        link: url,
+        description: `获取Binance数字货币及交易对上新公告`,
```
Comment (TonyRL) [inline lib/routes/binance/launchpool.ts:39]:
```suggestion
        description: '获取Binance数字货币及交易对上新公告',
```

File: lib/routes/binance/launchpool.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            source: ['binance.com/:lang/support/announcement'],
+        },
+    ],
+    name: 'Binance数字货币及交易对上新',
+    maintainers: ['zhenlohuang'],
+    handler,
+};
+
+async function handler() {
+    const baseUrl = `https://www.binance.com/zh-CN/support/announcement`;
```
Comment (TonyRL) [inline lib/routes/binance/launchpool.ts:22]:
```suggestion
    const baseUrl = 'https://www.binance.com/zh-CN/support/announcement';
```

File: lib/routes/binance/launchpool.ts [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+export const route: Route = {
+    path: '/launchpool',
+    categories: ['finance'],
+    example: '/binance/launchpool',
+    radar: [
+        {
+            source: ['binance.com/:lang/support/announcement'],
+        },
+    ],
+    name: 'Binance数字货币及交易对上新',
```
Comment (TonyRL) [inline lib/routes/binance/launchpool.ts:16]:
```suggestion
    name: '数字货币及交易对上新',
```

File: lib/routes/binance/launchpool.ts [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+    return {
+        title: 'Binance | 数字货币及交易对上新',
+        link: url,
+        description: `获取Binance数字货币及交易对上新公告`,
+        item: articles.map((item) => ({
+            title: item.title,
+            link: `${baseUrl}/${item.code}`,
+            description: item.title,
+            pubDate: timezone(parseDate(item.releaseDate), +8),
```
Comment (TonyRL) [inline lib/routes/binance/launchpool.ts:44]: Do not adjust the timezone of Unix timestamp.

**Comment by zhenlohuang** [comment]
It seems the `Test / Vitest on Node 21` failure is not related to my changes.

**Pull request merged**